src/dmd/cppmanglewin.d

```diff
@@ -167,7 +160,7 @@ public:
         if (checkImmutableShared(type, loc))
             return;
 
-        if (type.isConst() && ((flags & IS_NOT_TOP_TYPE) || (flags & IS_DMC)))
+        if (type.isConst() && (flags & IS_NOT_TOP_TYPE))
```

Comment:
The same clean-up of `IS_DMC` has been done in `dmd.cppmangledmc` as well, but the removal has been inverted.